sickrage/providers/GenericProvider.py

```diff
@@ -518,3 +519,18 @@ def _make_url(self, result):
 
     def _verify_download(self, file_name=None):  # pylint: disable=unused-argument,no-self-use
         return True
+
+    @property
+    def recent_results(self):
+        if sickbeard.provider_recent_results.get(self.get_id()):
+            return sickbeard.provider_recent_results.get(self.get_id())[::-1]
```

Comment:
Reverse the order, making sure that when looping the (newest/latest added) items are on top